examples/launch_height_baselines.py

```diff
@@ -36,26 +41,17 @@
     max_steps = 100
     n_workers = 4
 
-    config_space = {
```

Comment:
This is just a simple clean-up, avoiding all this copy and paste, same for all other `launch_XYZ.py` changes

Reply:
Can we please keep the inline definitions on the example? 

I believe importing constants from another file make them quite less readable for someone checking the library (for instance you cant know that RESOURCE_ATTR is a string without loading another file).

Reply:
OK, reverting back our examples to bad practices. I suppose cleaning up the unit tests is OK?

Reply:
Shall I add a comment:
"If you like to know where the strings "mean_loss" and "epoch" are coming from, just open the file whose name is hard-coded below, search for "report", and you'll find the names hard-coded there?"

And then copy and paste this comment across all `launch_xyz.py` files?

Reply:
I do not think this comment would help given that we already explain that report keys must be connected to the Tuner in the home page and I would expect this to be reasonably intuitive for users.

Reply:
OK, then let us keep it as it is.